alf/bin/train_play_test.py

```diff
@@ -415,6 +415,12 @@ def test_ac_simple_navigation(self):
             skip_checker=self._skip_if_socialbot_unavailable,
             extra_train_params=ON_POLICY_TRAIN_PARAMS)
 
+    # def test_sac_breakout(self):
```

Comment:
It might be good to add a note here on why it is disabled as reference for the future?

Reply:
This test case shouldn't relate to this PR, so there seems no reason for commenting it out.

Reply:
Haven't debugged, the current OFF_POLICY_TRAIN_PARAMS setup will result in not enough data error reported during training.  I've tried to increase collect steps to 8 and unroll length to 8, which fixed this problem, but it resulted in another test config to fail: during playing, test_dyna_actrepeat_sac_bipedal_walker: AssertionError: Conf file not found! Check your root_dir.

Added a comment there.

Reply:
> Haven't debugged, the current OFF_POLICY_TRAIN_PARAMS setup will result in not enough data error reported during training. I've tried to increase collect steps to 8 and unroll length to 8, which fixed this problem, but it resulted in another test config to fail: during playing, test_dyna_actrepeat_sac_bipedal_walker: AssertionError: Conf file not found! Check your root_dir.
> 
> Added a comment there.

The error might be because that you've changed `initial_collect_steps` and `unroll_length`? It's weird that sac_breackout_conf.py has this problem of not enough data, because all other sac test cases don't have. In fact, OFF_POLICY_TRAIN_PARAMS are supposed to change the configuration and make sure the training has no error for every off-policy algorithm? I think it's best to figure out why the current OFF_POLICY_TRAIN_PARAMS fails for sac_breakout_conf.py, but not for other sac related confs.

Reply:
I guess it might be because you've used `FrameStacker` in data transformer. Try setting the `stack_size=1` in `OFF_POLICY_PARAMS` and see how it goes.

Reply:
Good guess, FrameStacker is the reason!  Fixed.